great_expectations/validator/validator.py

```diff
@@ -72,6 +72,43 @@
 MAX_METRIC_COMPUTATION_RETRIES = 3
 
 
+ValidationStatistics = namedtuple(
+    "ValidationStatistics",
+    [
+        "evaluated_expectations",
+        "successful_expectations",
+        "unsuccessful_expectations",
+        "success_percent",
+        "success",
+    ],
+)
+
+
+def _calc_validation_statistics(validation_results):
+    """
+    Calculate summary statistics for the validation results and
+    return ``ExpectationStatistics``.
+    """
+    # calc stats
+    successful_expectations = sum(exp.success for exp in validation_results)
+    evaluated_expectations = len(validation_results)
+    unsuccessful_expectations = evaluated_expectations - successful_expectations
+    success = successful_expectations == evaluated_expectations
+    try:
+        success_percent = successful_expectations / evaluated_expectations * 100
+    except ZeroDivisionError:
```

Comment:
Can this try/except be replaced with an if statement? (if evaluated_expectations != 0)

Reply:
@donaldheppner Potentially -- I am not sure right now as it would need to go into a separate effort/PR (because I did not write this method -- I only moved it to the section of the module where all private methods are placed.  Thanks!

Reply:
okay